tests/skills/test_qraft_workflowspec.py

```
@@ -563,6 +563,49 @@ def test_definition_nonfinite_value_blocks(self, tmp_path, bad_value):
+        doc.update(overrides)
+        return doc
+
+    def test_a5_workflow_ref_version_missing(self):
+        validate = _load_module("qraft_validate", SKILL_DIR / "validate_run.py")
+        a, b = validate.semantic_check(self._run_doc(workflow_ref={}))
+        assert any(x.startswith("A5:") for x in a)
+        assert not any(x.startswith("A6:") for x in a)
+
+    def test_a6_request_prompt_missing(self):
+        validate = _load_module("qraft_validate", SKILL_DIR / "validate_run.py")
+        a, b = validate.semantic_check(self._run_doc(request={}))
```

Ruff (0.16.1)

[warning] Unpacked variable `b` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)